### PR TITLE
Fix pipenv installation

### PR DESCRIPTION
**Pull request by stephendonner**
@davehunt @tarekziade r?

This fixes https://github.com/mozilla/servicebook/issues/60

### CODE REVIEW

**Comment by stephendonner** [comment]
So...this fixes my local build in Docker, but not in Travis; I'll probably need some help with this, if either of you are up for it.

File: Dockerfile [hunk 1]
```diff
@@ -7,9 +7,10 @@ RUN addgroup --gid 10001 app
 RUN adduser --gid 10001 --uid 10001 --home /app --shell /sbin/nologin --no-create-home --disabled-password --gecos we,dont,care,yeah app
 
 WORKDIR /app
-RUN pip install pipenv uwsgi
+COPY Pipfile pipenv.txt /app/
+RUN pip install -r pipenv.txt uwsgi
 COPY . /app
-RUN pipenv install --deploy --system
+RUN pipenv install --deploy --system --skip-lock
```
Comment (davehunt) [inline Dockerfile]: Remove `--skip-lock`

File: pipenv.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+pipenv==11.8.3
```
Comment (davehunt) [inline pipenv.txt]: Add this to `.pyup.yml`

File: tox.txt [hunk 3]
```diff
@@ -0,0 +1 @@
+tox==2.9.1
```
Comment (davehunt) [inline tox.txt]: Add this to `.pyup.yml`

File: pipenv.txt (hunk 2 above)
Comment (stephendonner) [reply pipenv.txt]: Done

File: tox.txt (hunk 3 above)
Comment (stephendonner) [reply tox.txt]: Done

**Pull request merged**